akka-actor-tests/src/test/java/akka/actor/JavaAPI.java

```diff
@@ -194,14 +194,7 @@ public ActorWithConstructorParams(int d) {
     }
 
     public void onReceive(Object msg) {
```

Comment:
Looks like the tests mustBeAbleToCreateActorWIthBoxedAndUnBoxedConstructorParams and  mustBeAbleToCreateActorWIthConstructorParams are identical because of boxing. Is it worth to keep them both? I don't know, indeed.